src/App.tsx

```diff
@@ -17,18 +40,20 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter onFilter={setSort} />
```

Comment:
Please trim search value
![image](https://github.com/mate-academy/react_dynamic-list-of-todos/assets/54936044/43475cd1-63ce-4938-81a6-9421d2c12fb3)